framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3480,71 +3467,95 @@ public final Tree declarationFromElement(Element elt) {
    * Returns the current class type being visited by the visitor. The method uses the parameter only
```

Comment:
Should "the parameter" now refer to both of them, or still only one (if so, say which one).